PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,24 @@
+PULL REQUEST TEMPLATE
+======================
+
+
+### Description
+> Please include a summary of the change, relevant motivation and context. If possible and applicable, include before and after screenshots and a URL where the changes can be seen.
+    
+Check any of the following which apply:
+
+* [ ]  Breaking change (fix or feature that would cause existing functionality to not work as expected)
+* [ ]  This change requires a documentation update
+    * [ ]  I have made corresponding changes to the documentation
```

Comment:
I would propose to add a checkbox to encourage checking with `tox` as is (now) expected.

Reply:
(or under the checklist below, but then I would suggest to clarify *what* new warnings the author can expect: unit tests, docs, both, etc)

Reply:
People only have to check with tox if they change Python right? Would be good to make this clear

Reply:
done